.travis.yml

```diff
@@ -12,14 +12,14 @@ deploy:
     skip_cleanup: true
     script: make travis-latest-deploy
     on:
-      repo: redhat-cop/quay-openshift-registry-operator
+      repo: redhat-cop/quay-bridge-operator
```

Comment:
Fix `redhat-cop` here.

Reply:
noted, fixed here and in a couple other places